android/ANDROID-Flash.md

```diff
@@ -168,4 +168,4 @@ At this point, Android should automatically boot from the eMMC.
 
 ## Licence
 
-[![Creative Commons Licence](https://licensebuttons.net/l/by-sa/4.0/88x31.png)] (http://creativecommons.org/licenses/by-sa/4.0/)
\ No newline at end of file
+[![Creative Commons Licence](https://licensebuttons.net/l/by-sa/4.0/88x31.png)] (http://creativecommons.org/licenses/by-sa/4.0/)
```

Comment:
hmm.. can't tell what's the change in this line..

Reply:
I think it had a Windows CRLF and was automatically changed by `vim` to LF only.